kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx

```diff
@@ -134,12 +134,21 @@ describe('BrokersList Component', () => {
         }));
       });
 
-      it('renders empty table', async () => {
+      it('renders list of all brokers', async () => {
         renderComponent();
         expect(screen.getByRole('table')).toBeInTheDocument();
-        expect(
-          screen.getByRole('row', { name: 'Disk usage data not available' })
-        ).toBeInTheDocument();
+        expect(screen.getAllByRole('row').length).toEqual(3);
+      });
+      it('opens broker when row clicked', async () => {
+        renderComponent();
+        await act(() => {
```

Comment:
i think there with the new version of user event , you can automatically await the userEvent

Reply:
fixed